packages/labs/react/examples/preact/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "target": "ES2020",
+    "module": "ESNext",
+    "moduleResolution": "NodeNext",
+    "noEmit": true,
```

Comment:
Test is already explicitly setting `--noEmit`. Is this needed?

Reply:
This is actually default for vite projects as vite doesn't use `tsc` but `esbuild` for transpiling only. I would consider the `--noEmit` in the test script redundant but added for the sake of explicitly signaling we're running `tsc` for type checking only.

Reply:
Excellent!